### PR TITLE
[ADD] dead_mans_switch_client

### PR DESCRIPTION
**Pull request by hbrunn**
see https://github.com/OCA/server-tools/pull/297 for discussion

### CODE REVIEW

**Comment by gurneyalex** [comment]
:+1:

File: dead_mans_switch_client/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# © 2015 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Dead man's switch (client)",
+    "version": "1.0.0",
```
Comment (lmignon) [inline dead_mans_switch_client/__openerp__.py]: 6.0.1.0.0

**Comment by lmignon** [comment]
:+1:

File: dead_mans_switch_client/__openerp__.py (hunk 1 above)
Comment (pedrobaeza) [inline dead_mans_switch_client/__openerp__.py]: Better 6.1.1.0.0 :wink:

Comment (lmignon) [inline dead_mans_switch_client/__openerp__.py]: @pedrobaeza As always your are right ;-)

**Comment by hbrunn** [comment]
@lmignon @pedrobaeza thanks, done

**Comment by pedrobaeza** [comment]
:+1:

**Comment by lmignon** [comment]
thank you  @hbrunn

**Pull request merged**